Gems/ScriptCanvas/Code/Include/ScriptCanvas/AutoGen/ScriptCanvasAutoGenRegistry.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <string>
+#include <unordered_map>
```

Comment:
@onecent1101 I think you should add a GHI to switch to using AZStd containers if we get support for having the the AZ Allocators available on startup via @burelc-amzn  RFC for allocator availability: https://github.com/o3de/sig-core/pull/31